.github/actions/set-up/action.yml

```diff
@@ -0,0 +1,36 @@
+name: Free unused disk space
+description: Remove unused packages and files for Android build
+
+runs:
+  using: composite
+  steps:
+    - uses: nttld/setup-ndk@v1
+      id: setup-ndk
+      with:
+        ndk-version: r25c
+        link-to-sdk: true
+    - name: Setup Java
+      uses: actions/setup-java@v4
+      with:
+        distribution: 'temurin'
+        java-version: 17
+    - name: Setup Gradle
+      uses: gradle/actions/setup-gradle@v3
+    - run: |
+        sudo apt-get update --fix-missing
+        sudo apt-get remove -y '^dotnet-.*' '^llvm-.*' '^mysql-server-core-.*' '^postgresql-.*' \
+          azure-cli google-chrome-stable google-cloud-cli firefox powershell microsoft-edge-stable mono-devel
+        sudo apt-get purge docker-ce docker-ce-cli
+        sudo apt-get autoremove -y
+
+        sudo rm -rf /usr/share/dotnet/
+        sudo rm -rf /usr/share/swift
+        sudo rm -rf /usr/share/miniconda
+        sudo rm -rf /usr/local/graalvm/
+        sudo rm -rf /usr/local/.ghcup/
+        sudo rm -rf /usr/local/share/powershell
+        sudo rm -rf /usr/local/share/chromium
+        sudo rm -rf /usr/local/lib/node_modules
+        sudo rm -rf /var/lib/docker
+        sudo rm -rf /var/lib/apt/lists
```

Comment:
Every (almost) ```RUN```  or ```CMD```line that affects the file system in a docker file increases the number of layers. Therefore, I think using ```&&``` and squashing the lines is better.

Reply:
I missed this part 😅
Thank you :)